src/Model/Entity/Sentence.php

```diff
@@ -136,4 +137,9 @@ protected function _getPermissions()
             return null;
         }
     }
+    
+    protected function _getCurrentUserReview()
+    {
+        return CurrentUser::correctnessForSentence($this->id);
+    }
```

Comment:
Why not use a containment of the UsersSentences model from the Sentence model to get the correctness?